Sources/Linq2Acad/AcadDatabase.cs

```diff
@@ -850,11 +956,24 @@ public static AcadDatabase Open(string fileName, DwgOpenMode openMode, string pa
     /// <param name="keepOpen">True, if the database should be kept open after it has been used. False, if the database should be closed.</param>
     /// <returns>The AcadDatabase instance.</returns>
     private static AcadDatabase OpenInternal(string fileName, DwgOpenMode openMode, string password, bool keepOpen)
+    {
+      Database database = GetDatabase(fileName, openMode, password);
+      return new AcadDatabase(database, keepOpen);
+    }
+
+    /// <summary>
+    /// Provides access to the drawing database in the given file.
+    /// </summary>
+    /// <param name="fileName">The name of the drawing database to open.</param>
+    /// <param name="openMode">The mode in which the drawing database should be opened.</param>
+    /// <param name="password">The password for the darwing database.</param>
+    /// <returns>The Autocad Database instance.</returns>
+    private static Database GetDatabase(string fileName, DwgOpenMode openMode, string password)
```

Comment:
So, if I understood correctly, we could:
- remove the `openMode` parameter from `OpenInternal`, since it is now always called for read only;
- change the `openMode` parameter of `GetDatabase` to a boolean `readOnly` or `readWrite` and change the callers' signature accordingly;
- remove the `DwgOpenMode` enum.
Is this correct?